integrations/azure_ai_search/pyproject.toml

```diff
@@ -0,0 +1,164 @@
+[build-system]
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+[project]
+name = "azure-ai-search-haystack"
+dynamic = ["version"]
+description = 'Haystack 2.x Document Store for Azure AI Search'
+readme = "README.md"
+requires-python = ">=3.8,<3.13"
+license = "Apache-2.0"
+keywords = []
+authors = [{ name = "deepset", email = "info@deepset.ai" }]
+classifiers = [
+  "License :: OSI Approved :: Apache Software License",
+  "Development Status :: 4 - Beta",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: Implementation :: CPython",
+  "Programming Language :: Python :: Implementation :: PyPy",
+]
+dependencies = ["haystack-ai", "azure-search-documents>=11.5", "azure-identity", "torch>=1.11.0"]
```

Comment:
Why is `torch` needed?

Reply:
I think I added this while trying to fix lib support in python- 3.13. Will remove.